h2o-py/h2o/model/model_base.py

```diff
@@ -1713,7 +1713,15 @@ def score_history(self):
         """DEPRECATED. Use :meth:`scoring_history` instead."""
         return self.scoring_history()
 
+    def rule_importance(self):
+        """
+        Retrieve rule importances for a Rulefit model
 
+        :return: H2OTwoDimTable
+        """
+        if self._model_json["algo"] != "rulefit":
+            raise H2OValueError("This function is available for Rulefit models only")
+        return self._model_json["output"]['rule_importance']
```

Comment:
if it should be made available only for Rulefit, it's much better to add this accessor in `h2o-bindings/bin/custom/python/gen_rulefit.py`:

```
def class_extensions():
    def rule_importance(self):
        #logic goes here
extensions = dict(
    __class__=class_extensions
)
```

Reply:
will fix

Reply:
e0efd1a